Makefile

```diff
@@ -0,0 +1,32 @@
+.PHONY: docker-clean
+docker-clean:
+	# when docker runs as a demon the mounted folders have permissions and cannot be deleted by the current user
+	# so to avoid an annoying "sudo" step another docker container is created to get similar permissions
+	docker run \
+		--rm \
+		-v $(shell pwd)/pgosm-data:/app/output \
+		rustprooflabs/pgosm-flex \
+		bash -c "rm -rf /app/output/*"
+	rmdir pgosm-data|| echo "folder pgosm-data did not exist"
```

Comment:
I would prefer to see removing the output directory as a dedicated optional step instead of running by default.  Always removing output directory requires downloading the files from Geofabrik every time. That's network traffic they don't need to be serving, and even with the small 17MB size of DC that can take a couple minutes when the download is running slow.

Reply:
Hm right, it's very annoying. I have a project with the same problem and tried to store the input file in `git-lfs` but ended up consuming the free tier in a few days.

Maybe having a separate make target only to download the file to run manually could be a compromise to have a completely reproducible process? IIRC the script downloads it only if necessary, so mounting the folders separately and deleting only the output one could be another option

Reply:
Oh -- Can we copy the files over from `tests/data/` into the `/pgosm-data` subdirectory to avoid the DL entirely?  That would mean using this method for testing would never test the download portion, I'm ok with that.  It would make testing this faster in general.

I had forgotten I committed the PBF/MD5 to make testing the imported data possible.

https://github.com/rustprooflabs/pgosm-flex/tree/main/tests/data

Reply:
Ah, I didn't notice. Now I changed the Makefile to copy these files in the target folder (and change the name into "latest"), the script already did a check so it doesn't download them when already present.